endpoint.bal

```diff
@@ -355,10 +356,12 @@ public client class Client {
     # + contactGroupResourceName - Contact Group resource name
     # + resourceNameToAdd - Contact resource name to add
     # + resourceNameToRemove - Contact resource name to remove
-    # + return - json on success, else an `error`
-    @display {label: "Modify contacts in a Contact Group"}
-    isolated remote function modifyContactGroup(string contactGroupResourceName, string[]? resourceNameToAdd = (), string[]? resourceNameToRemove = ()) returns 
-                                       @tainted error? {
+    # + return - () on success, else an `error`
+    @display {label: "Modify Contacts In Contact Group"}
+    isolated remote function modifyContactGroup(@display {label: "Contact Group Resource Name"} string contactGroupResourceName, 
```

Comment:
Can we remove `Resource` and keep it as `Contact Group Name`. Does Resource tells anything?